Gemfile

```diff
@@ -68,6 +70,9 @@ group :development, :test do
   gem 'rubocop', require: false
   gem 'rubocop-performance', require: false
   gem 'rubocop-rails', require: false
+
+  gem 'irb'
```

Comment:
Apparently, `ubi9/ruby-31` image does not have these installed.

Also, I was testing the image built by the `Dockerfile` locally, and even when I install `rubygem-irb` and `rubygem-rdoc` dnf packages in the Dockerfile, rails can't start because it doesn't find them. Weird! Probably, I'm just doing something wrong.

Reply:
It is only the `rdoc` that creates the troubles. Looking into it.